apps/mux/src/index.tsx

```diff
@@ -1,61 +1,48 @@
-import * as React from 'react';
+import React from 'react';
 import { render } from 'react-dom';
-import { Note, Paragraph, Spinner, Button, TextLink } from '@contentful/forma-36-react-components';
-import { init, locations, AppExtensionSDK, FieldExtensionSDK } from '@contentful/app-sdk';
-import { createUpload } from '@mux/upchunk';
-import MuxPlayer from '@mux/mux-player-react';
-import '@contentful/forma-36-react-components/dist/styles.css';
-import './index.css';
 
-import Config from './config';
-import DeleteButton from './deleteButton';
-import ApiClient from './apiClient';
+import { init, locations, AppExtensionSDK, FieldExtensionSDK } from '@contentful/app-sdk';
+import {
+  Button,
+  Note,
+  Spinner,
+  TextLink,
+  Tabs,
+  Heading,
+  Box,
+  Flex,
+  Switch,
+} from '@contentful/f36-components';
+import { Form, FormControl, Checkbox, TextInput } from '@contentful/f36-forms';
+
+import MuxPlayer from '@mux/mux-player-react/dist/index.cjs';
+import MuxUploader from '@mux/mux-uploader-react/dist/index.cjs';
+import { MuxUploaderDrop } from '@mux/mux-uploader-react/dist/index.cjs';
```

Comment:
Doesn't this work?
```suggestion
import MuxPlayer from '@mux/mux-player-react';
import MuxUploader from '@mux/mux-uploader-react';
import { MuxUploaderDrop } from '@mux/mux-uploader-react';
```

Reply:
Unfortunately for the moment we have to use the cjs import. Once we can upgrade to React Scripts v5 then we can switch back.